pam/gdm/protocol_test.go

```diff
@@ -0,0 +1,687 @@
+package gdm
+
+import (
+	"bytes"
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func reformatJSON(t *testing.T, input []byte) []byte {
+	// We can't safely compare for JSON values when generated via protobuf
+	// so we initially pass it to native implementation to make it rebuild
+	// the JSON data so that the output is more reliable.
+	// See: https://protobuf.dev/reference/go/faq/#unstable-json
+	t.Helper()
```

Comment:
Can you move `t.Helper()` to the first line and scope it (i.e. empty line between it and the rest of the function code)? (Same thing for `t.Parallel()`)